package.json

```diff
@@ -28,13 +29,20 @@
     "@babel/core": "^7.18.10",
     "@babel/node": "^7.18.10",
     "@babel/preset-env": "^7.18.10",
+    "@types/reflect-metadata": "^0.1.0",
     "crossenv": "0.0.2-security",
     "csvtojson": "^2.0.10",
+    "dotenv": "^16.0.2",
     "express": "^4.18.1",
     "express-joi-validation": "^5.0.1",
     "joi": "^17.6.0",
     "nodemon": "^2.0.19",
+    "pg": "^8.8.0",
+    "reflect-metadata": "^0.1.13",
+    "sequelize": "^6.23.0",
```

Comment:
we have already used `typescript-sequelize`, so `sequelize` is useless in our project